docs: Document Ansible logging configuration

NicolasT opened the pull request:
Fixes: #255
See: https://github.com/scality/metalk8s/issues/255
See: RFE-152

NicolasT committed:
docs: Document Ansible logging configuration

Fixes: #255
See: https://github.com/scality/metalk8s/issues/255
See: RFE-152

Zempashi commented on .gitignore:
```diff
@@ -1,3 +1,10 @@
+# We have `ansible.cfg` in VCS, but because one may make local changes, we don't
```
Or part of a make shell, copy over an ansible.cfg.sample. Only the sample is checked in git and allow users to modify the ansible.cfg at will (or loose it if making another git clone)

NicolasT replied:
Yeah, that's a sensible approach (though I wouldn't let `make shell` create a copy, that's a bit IMHO).

wabernat committed:
Joined two sentence fragments, added a verb.

wabernat committed:
Copy edits

Changed first question to read in the correct case and mood.

Clarified wording of first paragraph.

Changed unordered list to ordered list.

wabernat committed:
Fixed run-on sentence

changed a few "whiches" to "thats"

wabernat reviewed (dismissed):
Approved for language, with edits.

NicolasT committed:
docs: Fixup sentence structure

As validated with tech-pubs.

See: 9c22a0b7e27c0ee911815211252df79b0bb813b1

NicolasT commented:
Needs new branch name.
